site/specs-temp/numbers.json

```diff
@@ -7515,6 +7515,292 @@
         }
       }
     },
+    "/accounts/{accountId}/importTnOrders/voice": {
+      "get": {
+        "tags": [
+          "/accounts"
+        ],
+        "description": "Retrieves the ImportVoiceTnOrder requests for the given account ID. A maximum of 1,000 orders can be retrieved per request. If no date range or specific query parameter (marked by <b class=\"required\">*</b> below) is provided, the order results will be limited to the last two years.\n",
```

Comment:
1. Why send 1000 be default for up to 2 years? Why not the requestor at least pass one of the criteria and then limit to 1000?
2. How do the requestor request next 1000 and so on or for that use `history` API?
3. @ajrice6713 Is the `/` needed for specific reason in tag names?

Reply:
The `/` shows in the documentation sidebar 

we have work scheduled later to go clean up those tags - its outside the scope of this PR to fix them now

Reply:
1, 2 it's how our generic history api works. we couldn't change it only for this order type

Reply:
@xxlaykxx I'm fine for 1. On 2, can you explain how the client will get next 1000 results and so on? i.e. is support for pagination needed?